### ISSUE TITLE
No line breaks in html emails

### ISSUE THREAD

**Issue by stephywells** [Strategy11/formidable-user-tracking#3]
When an html email is sent with the referrer info, there are no line breaks between urls. 

I haven’t confirmed this issue, or verified if it’s in Formidable or this addon.

**Comment by mswindler** [comment]
Reported again here: https://secure.helpscout.net/conversation/924685768/14003/

**Comment by AbdiTolesa** [comment]
@stephywells I just give this a shot and it looks to display with line breaks in place, not sure how to replicate this issue. Were you able to replicate it?
![image](https://user-images.githubusercontent.com/41271840/192010516-062e8fef-95a6-47fb-ab13-3e5bbc480baf.png)

**Comment by stephywells** [comment]
@AbdiTolesa in that screenshot from the customer, there isn't any referrer showing in there, so maybe that's part of it? Instead of spending time with the old code though, let's update this for the new code and make sure this isn't a problem.

**Comment by AbdiTolesa** [comment]
> @AbdiTolesa in that screenshot from the customer, there isn't any referrer showing in there, so maybe that's part of it? Instead of spending time with the old code though, let's update this for the new code and make sure this isn't a problem.

@stephywells With the new version, it would just include the referrer part (without the user journey) as we are using a different array key to store the user journey. Should we update lite to include the whole user journey or leave it as it is now?

**Comment by stephywells** [comment]
Yep, we need to update this to show the whole user journey. That part is included here:
https://github.com/Strategy11/formidable-user-tracking/issues/9

**Comment by AbdiTolesa** [comment]
@Crabcyborg I think html emails have line breaks in the next version to be released and this issue might be safe to close with that?
